rest-api/src/test/java/io/quarkus/qe/RepositoryResourceTest.java

```diff
@@ -35,15 +36,16 @@
 @QuarkusTestResource(H2DatabaseTestResource.class)
 public class RepositoryResourceTest {
 
-    private static final long AN_ENTITY_ID = 100;
+    private static final long DEFAULT_ENTITY_ID = 100;
+    private static final long NOT_FOUND_ENTITY_ID = 777;
```

Comment:
I don't understand these two fields, the id is extracted from entity.id if the entity has been created.